modules/imap/handler_modules.php

```diff
@@ -1304,6 +1314,20 @@ public function process() {
             $this->out('folder_status', $status);
             $this->out('imap_unread_data', $msg_list);
             $this->out('imap_server_ids', $form['imap_server_ids']);
+
+            $webhook_token = $this->user_config->get('webhook_token_setting');
+            $msg_count = count($msg_list);
+            $email_to = $msg_list[0]['to'];
+            if ($msg_count > 0) {
+                $interval = 5 * 60;
```

Comment:
5 must be configurable according to @marclaporte comment.

Reply:
Okay I shall apply changes. Thanks @Shadow243.